environment.yml error fix

Karwot opened the pull request:
Fixing the error of conda not being able to create the environment correctly:

CondaValueError: invalid package specification: setuptools <59.6.

ethanwhite reviewed (changes requested):
Thanks @Karwot. Since we're here can you remove the extra space before the `<` as well. Should be `setuptools<59.6` right?

Karwot commented:
> Thanks @Karwot. Since we're here can you remove the extra space before the `<` as well. Should be `setuptools<59.6` right?

Good catch :) I just removed the dot since that was causing the error for me but it's definitely better to keep one convention inside the file.
Updated.

ethanwhite commented:
Thanks @Karwot!
